showcase/package.json

```diff
@@ -120,8 +120,9 @@
     "@angular/platform-browser-dynamic": "~7.2.2",
     "@angular/platform-server": "~7.2.2",
     "@angular/router": "~7.2.2",
-    "@nationalbankbelgium/stark-core": "file:../dist/packages-dist/stark-core/nationalbankbelgium-stark-core-10.0.0-beta.2-ebdff291.tgz",
-    "@nationalbankbelgium/stark-ui": "file:../dist/packages-dist/stark-ui/nationalbankbelgium-stark-ui-10.0.0-beta.2-ebdff291.tgz",
+    "@mdi/angular-material": "^3.4.93",
```

Comment:
Can you remove this package? It should not be added to the showcase since it is already a dependency of `stark-ui`.